docs/samples/Create Asset.md

```diff
@@ -0,0 +1,295 @@
+                "type": "test-type",
+                "keyName": "test-key-name"
+              }
+            }' \
+    -s -o /dev/null -w 'Response Code: %{http_code}\n'
+```
+
+In the following chapters, only the contents of the request body will change in accordance to the asset payload.
+
+## 5. Custom Property Asset
+
+In the real world, an asset usually doesn't consist of the bare minimum but also contains custom properties, which store additional information.
+The basic use-case is that each property only consists of primitive datatypes and therefore simple key/value pairs.
+Those pairs are then stored inside the `properties` field.
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "edc:asset": {
+    "@type": "edc:asset",
+    "@id": "some-asset-id",
+    "properties": {
+      "name": "some-asset-name",
+      "description": "some description",
+      "edc:version": "0.2.1",
+      "contenttype": "application/json"
+    }
+  },
+  "edc:dataAddress": {
+    "@type": "edc:dataAddress",
+    "type": "test-type",
+    "keyName": "test-key-name"
+  }
+}
+```
+
+```bash
+curl -X POST "${CON_DATAMGMT_URL}/management/v2/assets" \
+    --header 'X-Api-Key: password' \
+    --header 'Content-Type: application/json' \
+    --data '{
+              "@context": {
+                "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+                "edc": "https://w3id.org/edc/v0.0.1/ns/"
+              },
+              "edc:asset": {
+                "@type": "edc:asset",
+                "@id": "some-asset-id",
+                "properties": {
+                  "name": "some-asset-name",
+                  "description": "some description",
+                  "edc:version": "0.2.1",
+                  "contenttype": "application/json"
+                }
+              },
+              "edc:dataAddress": {
+                "@type": "edc:dataAddress",
+                "type": "test-type",
+                "keyName": "test-key-name"
+              }
+            }' \
+    -s -o /dev/null -w 'Response Code: %{http_code}\n'
+```
+
+## 6. Private Property Asset
+
+A new addition are the private properties.
+Private properties will not be sent through the dataplane and are only accessible via the management API.
+This enables the storage of additional information pertaining the asset, that is not relevant for the consumer, but is nonetheless useful for the provider.
+Private properties are stores inside the `privateProperties` field.
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "edc:asset": {
+    "@type": "edc:asset",
+    "@id": "some-asset-id",
+    "properties": {
+      "name": "some-asset-name",
+      "description": "some description",
+      "edc:version": "0.2.1",
+      "contenttype": "application/json"
+    },
+    "edc:privateProperties": {
```

Comment:
why do we have here the `edc` prefix and on the "normale" properties not?

Reply:
Is it, because the edc stores everything which is prefixed with `edc:` separately?